### PR TITLE
[zh-cn]Update /docs/reference/kubectl/_index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
page preview: https://deploy-preview-34713--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/kubectl/
image

### CODE REVIEW

File: content/zh-cn/docs/reference/kubectl/_index.md [hunk 1]
```diff
@@ -34,9 +34,9 @@ files by setting the `KUBECONFIG` environment variable or by setting the
 [`--kubeconfig`](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) flag.
 -->
 `针对配置信息，`kubectl` 在 `$HOME/.kube` 目录中查找一个名为 `config` 的配置文件。
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/reference/kubectl/_index.md:36]:
```suggestion
针对配置信息，`kubectl` 在 `$HOME/.kube` 目录中查找一个名为 `config` 的配置文件。
```
多加的这个 "`" 符号应该才是链接渲染失败的原因。

File: content/zh-cn/docs/reference/kubectl/_index.md [hunk 2]
```diff
@@ -34,9 +34,9 @@ files by setting the `KUBECONFIG` environment variable or by setting the
...
-你可以通过设置 `KUBECONFIG` 环境变量或设置
-[`--kubeconfig`](/zh-cn/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
-参数来指定其它 [kubeconfig](/zh-cn/docs/concepts/configuration/organize-cluster-access-kubeconfig/) 文件。
+你可以通过设置 `KUBECONFIG` 环境变量或设置 
+`[`--kubeconfig`](/zh-cn/docs/concepts/configuration/organize-cluster-access-kubeconfig/)`
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/reference/kubectl/_index.md:38]: page preview: https://deploy-preview-34713--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/kubectl/

image

```suggestion
[`--kubeconfig`](/zh-cn/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
```

**Review by mengjiao-liu — CHANGES_REQUESTED** [review]

**Comment by yanrongshi** [comment]
Accidentally merged someone else's request, I will reopen a pr

**Comment by tengqm** [comment]
@yanrongshi You dont have to reopen a new PR again and again. Fix this PR at your local machine. Then `git commit --amend`, followed by `git push -f`. It is not hard, but you have to learn the right way to push PRs.

**Comment by yanrongshi** [comment]
> @yanrongshi You dont have to reopen a new PR again and again. Fix this PR at your local machine. Then `git commit --amend`, followed by `git push -f`. It is not hard, but you have to learn the right way to push PRs.

谢谢大佬，在git上的能力确实存在不足，这几天总算学会了，接下来不会再出现这种问题了，感谢

**Comment by tengqm** [comment]
Still working on this, @yanrongshi?

**Comment by yanrongshi** [comment]
> 还在做这个，@yanrongshi?

yes，today I will update this fire！

**Pull request closed without merging**